DM-23898: Update code based on testing with Tekniker's simulator

r-owen opened the pull request.

andrewbheyer commented:
![SimulationConnection](https://user-images.githubusercontent.com/31416633/80115438-bfce4400-8552-11ea-8d49-b4f8c48ae3e0.png)

r-owen commented:
bin/run_mock_tma.py:

Agreed. I removed the unnecessary "f"

bin/tma_commander.py:

port + 1: good suggestion. I have added an in-line comment before the 4 instances that I found.

CommandClass: good suggestion. I added new constant MTMount.commands.NUM_HEADER_FIELDS and use it here and in commands.py. I also added inline comments at the two points you mention to explain what is going on.

246 a very general Exception is caught: This code is actually correct. It prints the reason the loop has failed, and then lets the loop exit without raising an exception. Without this catch the loop will fail silently, and asyncio will probably complain at some point about an "unhandled exception".

mock/axis_device.py:

Fixed eanables -> enables. Good catch.

Regarding the comment on line 127: I believe the clock can only be non-monotonic when running under Docker on macOS (it's a bug). Docker on linux doesn't show the problem. I reworded the comment as follows, updating the three locations I found:
```
# Provide some slop for non-monotonic clocks, which are
# sometimes seen when running Docker on macOS.
```

The return on line 93 is indeed unnecessary. I removed it.

Log messages. My intent was to use lowercase when referencing a method or variable name (i.e. match the name exactly as it is spelled in the code). But I decided you are right and it would be just as clear to change "read_loop" to "Read loop" and so on. I think  I made log messages start with capital letters everywhere.
